### PR TITLE
[TS] LPS-199001

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, the previous pr is https://github.com/liferay-core-infra/liferay-portal/pull/5771

cc @wanderlast

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:friendly-url

**Comment by Kvale1733** [comment]
No regressions found.

File: modules/apps/portal-language/portal-language-extender/src/main/java/com/liferay/portal/language/extender/internal/LanguageResourcesExtender.java [hunk 1]
```diff
@@ -144,10 +144,13 @@ private void _registerResourceBundles(
 
 			String languageId = StringPool.BLANK;
 
-			if (urlPath.contains(StringPool.UNDERLINE)) {
+			index = StringUtil.indexOfAny(
```
Comment (shuyangzhou) [inline modules/apps/portal-language/portal-language-extender/src/main/java/com/liferay/portal/language/extender/internal/LanguageResourcesExtender.java:147]: @yuhai it is a single char searching, why `StringUtil.indexOfAny()` please keep it as plain String.indexOf() for char.

**Comment by shuyangzhou** [comment]
Please see comment above and resend, thanks!

**Pull request closed without merging**